client/src/components/Header.js

```diff
@@ -1,12 +1,27 @@
 import React from "react";
 import AppBar from "@mui/material/AppBar";
 import styled from "styled-components";
+import { Link } from "react-router-dom";
+import Add from "../pages/AddFood";
+import Details from "../pages/Details";
+import Home from "../pages/Landing";
 
 function Header() {
   return (
-    <AppBar position="static" style={{ height: "80px" }}>
+    <AppBar style={{ height: "90px", backgroundColor: "black" }}>
       <Container maxWidth="xl">
         <h1 className="u-text--center">Murial's Restaurant</h1>
+        <section>
+          <Link style={{ color: "white", fontSize: "15px" }} to="/home">
+            Landing |
+          </Link>
+          <Link style={{ color: "white", fontSize: "15px" }} to="/add">
+            AddFood |
+          </Link>
+          <Link style={{ color: "white", fontSize: "15px" }} to="/details">
+            Details |
+          </Link>
+        </section>
```

Comment:
It would be really cool to see styling implemented without inline styles.  What do you think is the ideal way to do that in your ecosystem?

Reply:
I thought `styled-components` but that was a bust :( ..I will keep researching and start a new branch for it

Reply:
Why not `styled-components`?

Reply:
Worked 💯